config/config.go

```diff
@@ -73,29 +73,29 @@ func LoadPseudonymsysUserSecret(user string) *big.Int {
 	return s
 }
 
-func LoadPseudonymsysOrgSecrets(org string) (*big.Int, *big.Int) {
+func LoadPseudonymsysOrgSecrets(orgName string) (*big.Int, *big.Int) {
 	m := viper.GetStringMap("pseudonymsys")
-	s1, _ := new(big.Int).SetString(m[org].(map[string]interface{})["s1"].(string), 10)
-	s2, _ := new(big.Int).SetString(m[org].(map[string]interface{})["s2"].(string), 10)
+	s1, _ := new(big.Int).SetString(m[orgName].(map[interface{}]interface{})["s1"].(string), 10)
+	s2, _ := new(big.Int).SetString(m[orgName].(map[interface{}]interface{})["s2"].(string), 10)
 	return s1, s2
 }
 
-func LoadPseudonymsysOrgPubKeys(org string) (*big.Int, *big.Int) {
+func LoadPseudonymsysOrgPubKeys(orgName string) (*big.Int, *big.Int) {
 	m := viper.GetStringMap("pseudonymsys")
-	h1, _ := new(big.Int).SetString(m[org].(map[string]interface{})["h1"].(string), 10)
-	h2, _ := new(big.Int).SetString(m[org].(map[string]interface{})["h2"].(string), 10)
+	h1, _ := new(big.Int).SetString(m[orgName].(map[interface{}]interface{})["h1"].(string), 10)
+	h2, _ := new(big.Int).SetString(m[orgName].(map[interface{}]interface{})["h2"].(string), 10)
 	return h1, h2
 }
 
-func LoadPseudonymsysCASecret(caName string) *big.Int {
+func LoadPseudonymsysCASecret() *big.Int {
 	m := viper.GetStringMap("pseudonymsys")
-	s, _ := new(big.Int).SetString(m[caName].(map[string]interface{})["D"].(string), 10)
+	s, _ := new(big.Int).SetString(m["ca"].(map[interface{}]interface{})["D"].(string), 10)
 	return s
 }
 
-func LoadPseudonymsysCAPubKey(caName string) (*big.Int, *big.Int) {
+func LoadPseudonymsysCAPubKey() (*big.Int, *big.Int) {
```

Comment:
I was unable to make `pseudonymsys_test` pass due to a panic stemming from lines like L98. On my workstation the string map keys are converted to lowercase, so `m["X"]` doesn't work for me (even though we specify "X" in the `default.yaml`), but `m["x"]` does work (perhaps its due to instability of viper library, not sure).

Also, viper supports a more elegant way of loading nested maps, please see the modified versions of functions that load pseudonymsys configuration below.

```go
func LoadPseudonymsysOrgSecrets(orgName string) (*big.Int, *big.Int) {
	org := viper.GetStringMap(fmt.Sprintf("pseudonymsys.%s", orgName))
	s1, _ := new(big.Int).SetString(org["s1"].(string), 10)
	s2, _ := new(big.Int).SetString(org["s2"].(string), 10)
	return s1, s2
}

func LoadPseudonymsysOrgPubKeys(orgName string) (*big.Int, *big.Int) {
	org := viper.GetStringMap(fmt.Sprintf("pseudonymsys.%s", orgName))
	h1, _ := new(big.Int).SetString(org["h1"].(string), 10)
	h2, _ := new(big.Int).SetString(org["h2"].(string), 10)
	return h1, h2
}

func LoadPseudonymsysCASecret() *big.Int {
	ca := viper.GetStringMap("pseudonymsys.ca")
	s, _ := new(big.Int).SetString(ca["d"].(string), 10)
	return s
}

func LoadPseudonymsysCAPubKey() (*big.Int, *big.Int) {
	ca := viper.GetStringMap("pseudonymsys.ca")
	x, _ := new(big.Int).SetString(ca["x"].(string), 10)
	y, _ := new(big.Int).SetString(ca["y1"].(string), 10)
	return x, y
}
```

So by using `viper.GetStringMap("pseudonymsys.<nestedMapName>")` we can avoid these very long type casts. Please update this (you can just replace the old functions with these).